backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -42,21 +58,32 @@
 @DisplayName("Resource /vaults")
 public class VaultResourceTest {
 
-	private final String vault1AdminJWT = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzM4NCIsInZhdWx0SWQiOiJ2YXVsdDEifQ.eyJpYXQiOjE1MTYyMzkwMTUsImV4cCI6NTgxNjIzOTAzMCwibmJmIjoxNTE2MjM5MDAwfQ.ibGVhY3WuD0nCE5tDaExH298JEEaJOEsLTUk1KKWW5kcCHgrl8lfDW4QktUn5O88Yu76Tvr47mxLDKbN1ZL3fHBbcW1QUv5KOw6tMlFWShlcPYN2qlZs1ety8mtOeH2_";
-	private final String vault2AdminJWT = "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCIsInZhdWx0SWQiOiJ2YXVsdDIifQ.eyJpYXQiOjE1MTYyMzkwMTUsImV4cCI6NTgxNjIzOTAzMCwibmJmIjoxNTE2MjM5MDAwfQ.l7Zvozwe3JTCOrxpscT5v1yZ5HALbWIFjtkSAJau3NLZGY8_ItX3PZmDfZrVGGOzahTtN1WQsGzDhtmHGoqoqfPITDLv_WcZKPomPI6Ch6R6lhXSC6m3gL81ZPH5w6R_";
+	private final String vault1AdminJWT;
+	private final String vault2AdminJWT;
 
 	@Inject
 	AgroalDataSource dataSource;
 
 	@Inject
 	Validator validator;
 
-
 	@BeforeAll
 	public static void beforeAll() {
 		RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
 	}
 
+	public VaultResourceTest() throws NoSuchAlgorithmException, InvalidKeySpecException {
+		var algorithmVault1 = Algorithm.ECDSA384((ECPrivateKey) getPrivateKey("MIG2AgEAMBAGByqGSM49AgEGBSuBBAAiBIGeMIGbAgEBBDAa57e0Q/KAqmIVOVcWX7b+Sm5YVNRUx8W7nc4wk1IBj2QJmsj+MeShQRHG4ozTE9KhZANiAASVL4lbdVoG9Wv0YpkafXf31YNN3rVD1/BAyZm4EYBg92X+taTvTlBjpaGWZuiSYRW9r+YQdKg1D3zAWb0UEKrOHjkgZ38MbBnTheGLlqH7VspuRWG12zydm0dF1ImiRik="));
+		var algorithmVault2 = Algorithm.ECDSA384((ECPrivateKey) getPrivateKey("MIG2AgEAMBAGByqGSM49AgEGBSuBBAAiBIGeMIGbAgEBBDCAHpFQ62QnGCEvYh/pE9QmR1C9aLcDItRbslbmhen/h1tt8AyMhskeenT+rAyyPhGhZANiAAQLW5ZJePZzMIPAxMtZXkEWbDF0zo9f2n4+T1h/2sh/fviblc/VTyrv10GEtIi5qiOy85Pf1RRw8lE5IPUWpgu553SteKigiKLUPeNpbqmYZUkWGh3MLfVzLmx85ii2vMU="));
+
+		vault1AdminJWT = JWT.create().withHeader(Map.of("vaultId", "vault1")).withIssuedAt(Instant.now()).sign(algorithmVault1);
+		vault2AdminJWT = JWT.create().withHeader(Map.of("vaultId", "vault2")).withIssuedAt(Instant.now()).sign(algorithmVault2);
+	}
+
+	private PrivateKey getPrivateKey(String keyBytes) throws NoSuchAlgorithmException, InvalidKeySpecException {
+		return KeyFactory.getInstance("EC").generatePrivate(new PKCS8EncodedKeySpec(Base64.getDecoder().decode(keyBytes)));
+	}
+
```

Comment:
1. use a `@Before public void setup()` method rather than a constructor during tests.
2. it is a bad idea to use `Instant.now()` during tests, as the test is no longer deterministic. Use hard-coded time stamps and make the validator accept a specific date which should be considered as "now"